lib/modules/onc_program/bulk_data_group_validation_sequence.rb

```diff
@@ -230,7 +273,13 @@ def get_file(file, use_token: true)
           )
         end
 
-        test_output_against_profile('Patient')
+        must_supports = [
+          {
+            profile: nil,
+            must_support_info: Inferno::Sequence::USCore310PatientSequence::MUST_SUPPORTS.dup
```

Comment:
This depends on the fact that these files were loaded first.  Are we sure they are loaded first?

Reply:
@czh-orz @yunwwang I'm kicking this off into another ticket for us to consider how we really want these tests to be organized.  I'm concerned about mixing stuff across modules and the type of knowledge you would need to know about what depends on what, etc.  But I'd like to get this in to fill the MUST SUPPORT gap for now.